### PR TITLE
OP-31: view_confidence — enforce the side-angle requirement

### PR DESCRIPTION
**Pull request by m-nweke**
This pull request adds a new metric for assessing whether a photograph was taken from the side or the front, and introduces a corresponding confidence factor for interpreting sagittal (side-view) posture measurements. The main additions include the implementation of the `view_confidence` and `view_confidence_factor` functions, their integration into the metrics package, and comprehensive tests to ensure correct behavior.

**New metric for camera angle assessment:**

* Added `view_confidence` function in `view.py` to compute the ratio of shoulder span to torso length in image space, which estimates whether the photo was taken from the side or the front. This provides a way to assess the reliability of posture measurements based on camera angle.
* Added `view_confidence_factor` function in `view.py` to produce a confidence score (between 0.4 and 1.0) for sagittal measurements, scaling with how close the camera angle is to a true side view.
* Updated `__init__.py` to export `view_confidence` and `view_confidence_factor` as part of the public metrics API. [[1]](diffhunk://#diff-d360466325dac39b7472e2e71c26f54bcfccdadc9913f4dde9d5cc786702057fR20) [[2]](diffhunk://#diff-d360466325dac39b7472e2e71c26f54bcfccdadc9913f4dde9d5cc786702057fR29-R30)

**Testing and validation:**

* Added a comprehensive test suite in `test_view.py` to verify the correctness of the new metric and its confidence factor, including handling of edge cases (e.g., missing keypoints, ambiguous geometry, and the full range of camera angles).

### CODE REVIEW

File: packages/posture-core/src/posture_core/metrics/view.py [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+the model's weakest dimension. So this metric produces a **confidence factor** the report applies
+to sagittal findings, rather than a veto that discards them. A lower-confidence finding is more
+useful to a user than a missing one, and more honest than a confident one.
+"""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Final
+
+from posture_core.geometry import distance, image_vec, midpoint
```
[the remark this answers is not in the document]
Comment (m-nweke) [reply packages/posture-core/src/posture_core/metrics/view.py]: Agreed, and it is the same defect `arms_crossed` had — fixed there in ff1bbd6 and missed here. Now `torso < MIN_LENGTH`, with the reasoning in a comment beside it.

The damage is worse in this metric than in `arms_crossed`, and only shows up face-on. Side-on the shoulders coincide, so the numerator is zero and a degenerate torso still yields `0.0` — the right answer by luck. Face-on the numerator is real: at a 1e-11 m torso the ratio measured **3.8e10**, which reads as "photographed from the front" against any threshold. That does not stay local. The ratio feeds `view_confidence_factor`, so a single degenerate torso pins *every* sagittal finding in the report to the 0.4 confidence floor. Both views are now in the parametrised test for exactly that reason.

One thing worth recording rather than glossing, since it slightly qualifies the fix: `MIN_LENGTH` is 1e-9 and this metric divides in **image** space, not metres. So the guard trips at roughly 1e-12 m of world torso, and a 5e-12 m torso — physically absurd — still measures. That is the constant doing its actual job, which is stopping a division by nothing, not judging whether a body is a plausible size. I kept the shared constant rather than introducing a pixel-scaled epsilon here: two epsilons either side of the same division is precisely the duplicate that drifts the first time one is retuned. There is a test pinning that boundary so the limitation is documented rather than assumed.

File: packages/posture-core/src/posture_core/metrics/__init__.py [hunk 2]
```diff
@@ -17,6 +17,7 @@
 from posture_core.metrics.head import craniovertebral_angle_deg
 from posture_core.metrics.knees import knee_flexion_deg
 from posture_core.metrics.trunk import trunk_inclination_deg
+from posture_core.metrics.view import view_confidence, view_confidence_factor
```
[the remark this answers is not in the document]
Comment (m-nweke) [reply packages/posture-core/src/posture_core/metrics/__init__.py]: Agreed. The module docstring states that everything here is `(resolver, thresholds) -> Metric` and that `report.py` runs the list without knowing anything about any entry; `view_confidence_factor` takes a ratio and returns a float, so exporting it put something in the namespace that the paragraph above says is not in it.

Removed from both the import and `__all__`, with a comment recording why it is deliberately absent — otherwise it reads as an oversight and gets "fixed" back in.

Nothing depended on the re-export. The only two callers, `test_view.py` and `report.py` on the OP-32 branch, already reach it as `posture_core.metrics.view.view_confidence_factor`; `report.py` imports the metric modules rather than the package namespace.

File: packages/posture-core/tests/test_view.py [hunk 3]
```diff
@@ -0,0 +1,131 @@
+"""Tests for `view_confidence` — the check the original app never performed."""
```
[the remark this answers is not in the document]
Comment (m-nweke) [reply packages/posture-core/tests/test_view.py:1]: Leaving this one as "the check".

The definite article is load-bearing. The original application *told* users the photograph had to be taken from a side angle and then assessed whatever arrived — so there is one specific check its own instruction implied and it never performed, and that is what this module supplies. "A check the original app never performed" would be true of any number of checks it also never performed, which is a weaker and vaguer claim than the one intended.
